frontend/templates/views/partials/header.j2

```diff
@@ -8,9 +8,15 @@
 {% if doc.pod_path.startswith(search_path) %}ap-m-nav-link-active{% endif %}
 {%- endmacro %}
 
+{# Simply set to False to disable banner #}
+{% set show_banner = True %}
+{% set banner %}
+{% include '/views/partials/banner.j2' %}
```

Comment:
This always includes (and loads) the banner template even if it will not be shown.

Reply:
You absolutely right, fixed this with an if condition.
Also I forgot changing show_banner back to False.